MANIFEST.in

```diff
@@ -4,4 +4,9 @@ include HISTORY.rst
 include LICENSE
 include README.rst
 include requirements.txt
-recursive-include djangocms_page_meta *.html *.png *.gif *js *jpg *jpeg *svg *py *po *mo
\ No newline at end of file
+recursive-include */media *
+recursive-include */boilerplates *
+recursive-include */templates *
+recursive-include */static *
+recursive-include */locale *
```

Comment:
Let's be less generic :) The strictier we are in packages the less cruft we ship in them